bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/PlugwiseHABindingConstants.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
```

Comment:
Update copyright year. Can be done with: `mvn license:format -pl :org.openhab.binding.plugwiseha` in root.

Reply:
When I run that command in the `openhab2-addons` root folder I keep getting this error: `[ERROR] Could not find the selected project in the reactor: :org.openhab.binding.plugwiseha @`

Any ideas as to how I fix this?